demoproj/settings.py

```diff
@@ -101,19 +104,22 @@
     'disable_existing_loggers': False,
     'filters': {
         'correlation_id': {
-            '()': 'django_guid.log_filters.CorrelationId'  # <-- Add correlation ID to the filters
+            '()': 'django_guid.log_filters.CorrelationId',  # <-- Add correlation ID to the filters
+        },
+        'celery_referrer': {
+            '()':'django_guid.celery.log_filters.CeleryReferralId'  # <-- Add depth to celery logs'
         }
     },
     'formatters': {
         'medium': {
-            'format': '%(levelname)s %(asctime)s [%(correlation_id)s] %(name)s %(message)s'  # <-- Format the log string
+            'format': '[%(correlation_id)s] [%(celery_referrer)s] %(name)s -- %(message)s'  # <-- Format the log string
```

Comment:
Removed log-level and time to make it easier to see the new log filter, but should remember to add this back in before merging